uitools/examples/cpp_quick/cpp_quick.pro

```diff
@@ -43,11 +43,11 @@ include($$PWD/../../toolkitcpp.pri)
 INCLUDEPATH += $$PWD/src
 
 HEADERS += \
-    src/BaseDemoModel.h
+    src/BaseDemol.h
```

Comment:
type here

Reply:
Ack. Litreally must have occured before commit because it tested fine. Good catch!